mlflow/sklearn/__init__.py

```diff
@@ -557,6 +557,8 @@ def autolog():
 
     **Example**
 
+    `See more examples <https://github.com/mlflow/mlflow/blob/master/examples/sklearn_autolog>`_
+
```

Comment:
The link looks like this:

![Screen Shot 2020-08-25 at 12 48 19](https://user-images.githubusercontent.com/17039389/91120725-4ea1c780-e6d1-11ea-818d-b929ca403e0d.png)

https://682-136202695-gh.circle-artifacts.com/0/docs/build/html/python_api/mlflow.sklearn.html